global-script.lua

```diff
@@ -2030,357 +2029,232 @@ scaleFactors = {
     [true]={name = "Large", position = 1.09, size = 1.1},
     [false]={name = "Standard", position = 1, size = 1},
 }
-posMap = {
+boardLayouts = {
     { -- 1 Board
         ["Balanced"] = {
-            Vector(5.96, 1.08, 16.59),
+            { pos = Vector(5.96, 1.08, 16.59), rot = Vector(0.00, 180.00, 0.00) },
         },
         ["Thematic"] = {
-            Vector(-1.93, 1.08, 20.44), -- NE
+            { pos = Vector(-1.93, 1.08, 20.44), rot = Vector(0.00, 180.00, 0.00), board = "NE" },
         },
     },
     { -- 2 Board
         ["Balanced"] = {
-            Vector(9.13, 1.08, 25.29),
-            Vector(0.29, 1.08, 10.21),
+            { pos = Vector(9.13, 1.08, 25.29), rot = Vector(0.00, 180.00, 0.00) },
+            { pos = Vector(0.29, 1.08, 10.21), rot = Vector(0.00, 0.00, 0.00) },
         },
         ["Thematic"] = {
-            Vector(9.54, 1.08, 18.07), -- E
-            Vector(-10.34, 1.08, 18.04), -- W
+            { pos = Vector(9.54, 1.08, 18.07), rot = Vector(0.00, 180.00, 0.00), board = "E" },
+            { pos = Vector(-10.34, 1.08, 18.04), rot = Vector(0.00, 180.00, 0.00), board = "W" },
         },
         ["Top to Top"] = {
-            Vector(9.13, 1.08, 25.29),
-            Vector(-0.06, 1.08, 9.62),
+            { pos = Vector(9.13, 1.08, 25.29), rot = Vector(0.00, 0.00, 0.00) },
+            { pos = Vector(-0.06, 1.08, 9.62), rot = Vector(0.00, 180.00, 0.00) },
         },
         ["Coastline"] = {
-            Vector(20.38, 1.08, 9.96),
-            Vector(2.54, 1.08, 10.34),
+            { pos = Vector(2.54, 1.08, 10.34), rot = Vector(0.00, 240.69, 0.00) },
+            { pos = Vector(20.38, 1.08, 9.96), rot = Vector(0.00, 240.69, 0.00) },
         },
         ["Opposite Shores"] = {
-            Vector(-4.22, 1.08, 18.91),
-            Vector(13.78, 1.08, 19.09),
+            { pos = Vector(-4.22, 1.08, 18.91), rot = Vector(0.00, 180.00, 0.00) },
+            { pos = Vector(13.78, 1.08, 19.09), rot = Vector(0.00, 0.00, 0.00) },
         },
         ["Fragment"] = {
-            Vector(-5.20, 1.08, 18.87),
-            Vector(10.12, 1.08, 19.08),
+            { pos = Vector(-5.20, 1.08, 18.87), rot = Vector(0.00, 90.00, 0.00) },
+            { pos = Vector(10.12, 1.08, 19.08), rot = Vector(0.00, 330.00, 0.00) },
         },
-        ["Inverted Fragment"] = {
-            Vector(-5.44, 1.08, 18.99),
-            Vector(10.12, 1.08, 19.08),
+        ["Fragment 2"] = {
```

Comment:
this should stay as `Inverted Fragment` there's code in json load to auto convert Fragment 2 into Inverted Fragment